File: segundo-semestre/lab121/c_sharp/tablas_de_multiplicar.cs
class MainClass {
  public static void Main(string[] args) {
    int a;
    int b;
    int c;

    for (a = 1; a <= 10; a++) {
      System.Console.WriteLine("Tabla: " + a);

      for (b = 1; b <= 10; b++) {
        c = a * b;
        System.Console.WriteLine(a + " x " + b + " = " + c);
      }
    }
  }
}

File: segundo-semestre/lab121/tarea1/java/tablas_de_multiplicar.java
class Main {
  public static void main(String[] args) {
    int a;
    int b;
    int c;

    for (a = 1; a <= 10; a++) {
      System.out.println("Tabla: " + a);

      for (b = 1; b <= 10; b++) {
        c = a * b;
        System.out.println(a + " x " + b + " = " + c);
      }
    }
  }
}
